stable/meteor/templates/cronjob.yaml

```diff
@@ -33,6 +33,20 @@ spec:
               - -c
               - |
                 meteor run /opt/recipes;
+                METEOR_EXIT_CODE=$?
+                echo "Meteor command exited with code METEOR_EXIT_CODE"
+
+                if [ $METEOR_EXIT_CODE -eq 0 ]; then
+                  {{- if .Values.cleanupStaleAssets.enabled }}
```

Comment:
does it mean we will not do anything (not even writing message) when cleanup is not enabled?

Reply:
yap, I create it so it is more flexible and easier when we need to disable the auto clean up assets for a certain reason

Reply:
it was just confirmation, whether we want to add message or something here. if it is expected to have nothing to do, then all good.